src/feed/feed.controller.ts

```
@@ -0,0 +1,20 @@
+import { Controller, Post, Param, Request, Get } from '@nestjs/common';
+import { FeedService } from './feed.service';
+ 
+
```

ESLint

[error] Delete `·⏎`

(prettier/prettier)